<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGUI</class>
 <widget class="QWidget" name="UserGUI">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>555</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UserGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 52, 52);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="dogPhoto">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 170, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="1">
            <widget class="QLineEdit" name="ageShow_2">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
             </property>
             <property name="text">
              <string>Breed:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="breedShow">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nameShow">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="ageShow">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
             </property>
             <property name="text">
              <string>URL:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
             </property>
             <property name="text">
              <string>Age:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="adoptBtn">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
background-color: rgb(0, 170, 0);</string>
             </property>
             <property name="text">
              <string>Adopt</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="nextBtn">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Century&quot;;
background-color: rgb(0, 170, 0);</string>
             </property>
             <property name="text">
              <string>Next</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QLineEdit" name="ageFilter">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="breedFilter">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
         </property>
         <property name="text">
          <string>Breed filter</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Century&quot;;
color: rgb(0, 170, 0);</string>
         </property>
         <property name="text">
          <string>Age filter</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Century</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 170, 0);</string>
         </property>
         <property name="text">
          <string>Adoption List</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 170, 0);
font: 12pt &quot;Century&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPushButton" name="viewBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 0);
font: 12pt &quot;Century&quot;;</string>
         </property>
         <property name="text">
          <string>View</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="openBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 0);
font: 12pt &quot;Century&quot;;</string>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
